packages/target-electron/build/afterPackHook.mjs

```diff
@@ -186,3 +185,49 @@ async function deleteNotNeededPrebuildsFromUnpackedASAR(
     )
   }
 }
+
+async function setFuses(context, asar) {
+  // Apply security fuses for all builds
+  let appPath
+  let executableName = context.packager.executableName ?? 'DeltaChat'
+  if (process.env.IS_PREVIEW) {
+    executableName = executableName + '-DevBuild'
+  }
+  switch (context.electronPlatformName) {
+    case 'darwin':
+    case 'mas':
+      appPath = `${context.appOutDir}/${executableName}.app`
+      break
+    case 'win32':
+      appPath = `${context.appOutDir}/${executableName}.exe`
+      break
+    default:
+      appPath = `${context.appOutDir}/${context.packager.executableName ?? 'deltachat-desktop'}`
+      break
+  }
+
+  if (!existsSync(appPath)) {
+    const files = await readdir(context.appOutDir)
+    
+    // Log the list of file names
+    console.log(`Files in context.appOutDir (${context.appOutDir}):`);
+    files.forEach(file => {
+      console.log(file);
+    });
+    throw new Error('Could not apply electron fuses since target not exists: ' + appPath)
+  }
+
+  console.log('Applying electron fuses to:', appPath)
+  await flipFuses(appPath, {
+    version: FuseVersion.V1,
+    [FuseV1Options.RunAsNode]: false, // Disables ELECTRON_RUN_AS_NODE
+    [FuseV1Options.EnableCookieEncryption]: false, // not needed

```

Comment:
> > EnableCookieEncryption
>
> probably not useful since we store no cookies, unless it also applies to localstorage.

How about WebXDC?

Also, why even mess with this fuse if it's not related to that supposed security issue and is disabled by default anyway? To me it seems better just to use the default value, i.e. not specify this here at all.

The same goes for `GrantFileProtocolExtraPrivileges`, and maybe other fuses.

Reply:
Of course we could just skip the fuses, if we don't override the defaults but to me it seems more transparent to write all settings explicit here